modules/nw-ingress-gateway-api-enable.adoc

```diff
@@ -0,0 +1,65 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-enable_{context}"]
+= Enabling Gateway API for the Ingress Operator
+
+When you enable Gateway API, it installs the subscription, the Istio operator, the custom resource defintions (CRDs), and creates the Istio resources. The following procedure describes how to enable Gateway API.
+
+.Procedure
+
+. Create a `GatewayClass` object with the following manifest:
++
+[source,yaml]
+----
+apiVersion: gateway.networking.k8s.io/v1
+kind: GatewayClass
+metadata:
+  name: openshift-default
+spec:
+  controllerName: openshift.io/gateway-controller/v1 <1>
+----
+<1> The controller name must be `v1` for the Ingress Operator to manage it. Setting this field to anything else will be ignored by the Ingress Operator, because it is assumed to be a different implementation.
+
+. Create a secret using the default certificate:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress create secret tls gwapi-wildcard --cert=wildcard.crt --key=wildcard.key
+----
+
+. Create a `Gateway` object with the following manifest:
++
+[source,yaml]
+----
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: example-gateway
+  namespace: openshift-ingress
+spec:
+  gatewayClassName: openshift-default <1>
+  listeners: <2>
+  - name: http
+    hostname: "*.gwapi.example.com"
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
```

Comment:
Please remove this section.  A single listener will be a better example to start with.

PS - can you tell me where you got this example from?